### PR TITLE
Add Ionic tutorials

### PR DESCRIPTION
**Pull request by azeey**
Part of https://github.com/gazebo-tooling/release-tools/issues/1027

## Summary
This copies the `harmonic` tutorials into the `ionic` directory and updates instances of Harmonic to Ionic. This also updates the library versions that have been bumped for Ionic. The best way to review is probably to do the diff between the `harmonic` and `ionic` directories.

## Checklist
- [x] Signed all commits for DCO
- [x] Updated documentation (as needed)

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: ionic/GUI_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+```bash
+8.0.0
+7.0.0
+6.0.0
+```
+
+To ensure you're working with Ionic, include the `--force-version` directive when starting up:
+
+```bash
+gz sim --force-version 8.0.0 shapes.sdf
```
Comment (iche033) [inline ionic/GUI_tutorial.md:31]:
```suggestion
gz sim --force-version 9.0.0 shapes.sdf
```

File: ionic/install.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+|   gz-sim           |       9.x     |
+|   gz-gui           |       9.x     |
+|   gz-launch        |       8.x     |
+|   gz-math          |       8.x     |
+|   gz-msgs          |      11.x     |
+|   gz-physics       |       8.x     |
+|   gz-plugin        |       3.x     |
+|   gz-rendering     |       9.x     |
+|   gz-sensors       |       9.x     |
+|   gz-tools         |       3.x     |
```
Comment (iche033) [inline ionic/install.md:45]:
```suggestion
|   gz-tools         |       2.x     |
```

File: ionic/install_osx_src.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+```
+
+General dependencies:
+
+```bash
+brew install assimp boost bullet cmake cppzmq dartsim@6.10.0 doxygen eigen fcl ffmpeg flann freeimage freetype gdal gflags google-benchmark gts ipopt jsoncpp libccd libyaml libzzip libzip nlopt ode open-scene-graph ossp-uuid ogre1.9 ogre2.3 pkg-config protobuf qt@5 qwt-qt5 rapidjson ruby tbb tinyxml tinyxml2 urdfdom zeromq
+```
+
+`dartsim@6.10.0` and `qt@5` are not sym-linked. To use those dependencies when building
+`gz-physics7` and `gz-gui8`, run the following after installation:
```
Comment (iche033) [inline ionic/install_osx_src.md:94]:
```suggestion
`gz-physics8` and `gz-gui9`, run the following after installation:
```

File: ionic/install_windows_src.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+```bash
+colcon graph
+```
+
+`colcon graph` should list the Gazebo libraries with an
+[interdependency diagram](https://colcon.readthedocs.io/en/released/reference/verb/graph.html#example-output).
+If that is the case, then you are ready to build the whole set of libraries:
+
+```bash
+colcon build --cmake-args -DBUILD_TESTING=OFF --merge-install --packages-up-to gz-sim8 gz-tools2
```
Comment (iche033) [inline ionic/install_windows_src.md:107]:
```suggestion
colcon build --cmake-args -DBUILD_TESTING=OFF --merge-install --packages-up-to gz-sim9 gz-tools2
```

File: ionic/install_windows_src.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+> ```
+> Remember to set the same partition in all other consoles.
+
+### Gazebo GUI workaround
+
+Although running `gz sim` without arguments is not supported on Windows,
+and `gz sim -g` is also not supported, there is a workaround you can apply
+to be able to launch `gz sim -g` on Windows.
+
+> Manually comment [these lines](https://github.com/gazebosim/gz-sim/blob/gz-sim7_7.5.0/src/cmd/cmdsim.rb.in#L497-L501) and [these lines](https://github.com/gazebosim/gz-sim/blob/gz-sim7_7.5.0/src/cmd/cmdsim.rb.in#L558-L562) in file `install\lib\ruby\gz\cmdsim8.rb`.
```
Comment (iche033) [inline ionic/install_windows_src.md:166]:
```suggestion
> Manually comment [these lines](https://github.com/gazebosim/gz-sim/blob/gz-sim7_7.5.0/src/cmd/cmdsim.rb.in#L497-L501) and [these lines](https://github.com/gazebosim/gz-sim/blob/gz-sim7_7.5.0/src/cmd/cmdsim.rb.in#L558-L562) in file `install\lib\ruby\gz\cmdsim9.rb`.
```

File: ionic/GUI_tutorial.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+
+Start by getting Gazebo up and running with a sample world:
+
+```bash
+gz sim shapes.sdf
+```
+
+`gz sim` will run the latest version of Gazebo you have installed.
+You can check if you have more than one version of Gazebo installed by running `gz sim --versions`.
+For example, if you have Ionic (8.x.x), Garden (7.x.x), and Fortress (6.x.x) installed, `--versions` might return:
```
[the remark this answers is not in the document]
Comment (azeey) [reply ionic/GUI_tutorial.md]: https://github.com/gazebosim/docs/pull/405/commits/1d0f9928f8b848adb6ca97aab899bd91bbf4a913

File: ionic/GUI_tutorial.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+
+```bash
+8.0.0
+7.0.0
+6.0.0
```
[the remark this answers is not in the document]
Comment (azeey) [reply ionic/GUI_tutorial.md:26]: https://github.com/gazebosim/docs/pull/405/commits/1d0f9928f8b848adb6ca97aab899bd91bbf4a913

File: ionic/GUI_tutorial.md (hunk 1 above)
Comment (azeey) [reply ionic/GUI_tutorial.md]: https://github.com/gazebosim/docs/pull/405/commits/1d0f9928f8b848adb6ca97aab899bd91bbf4a913

File: ionic/install.md (hunk 2 above)
Comment (azeey) [reply ionic/install.md]: https://github.com/gazebosim/docs/pull/405/commits/1d0f9928f8b848adb6ca97aab899bd91bbf4a913

File: ionic/install_osx_src.md (hunk 3 above)
Comment (azeey) [reply ionic/install_osx_src.md]: https://github.com/gazebosim/docs/pull/405/commits/1d0f9928f8b848adb6ca97aab899bd91bbf4a913

File: ionic/install_windows_src.md (hunk 5 above)
Comment (azeey) [reply ionic/install_windows_src.md]: https://github.com/gazebosim/docs/pull/405/commits/1d0f9928f8b848adb6ca97aab899bd91bbf4a913 but I noticed that this is using the `gz-sim7-7.5.0` tag. We'll probably need to update it once we create a stable branch.

**Review by iche033 — APPROVED** [review]

**Review by mjcarroll — APPROVED** [review]

**Pull request merged**